.github/workflows/pr-checklist.yaml

```diff
@@ -0,0 +1,35 @@
+# alt: We could have used the .github/pull_request_template.md file
+# for this checklist, but that means that the checklist will clutter
```

Comment:
A bit more context that you can add (rephrasing as you wish). Initially the commit description was the default for "squash and merge", which is made of all the commit messages that you wrote for that PR. But people writes very bad commit messages, "init", "progress", "more work", etc. So while our PR descriptions were pretty good, our Git log looked pretty bad. Then we tried a GitHub option that lets you use the PR title+description as the commit title+message, and I believe everyone appreciated it. However we had the checklist adding noise to every commit message, to remove it you had to do it manually at the time of merging. This PR will fix that, so we can have a nice Git history.

Reply:
Done.